plugins/metrics/scripts/send_session_metrics.py

```
@@ -0,0 +1,344 @@
+                message = entry.get('message', {})
+                if isinstance(message, dict):
+                    content = message.get('content', [])
+                    if isinstance(content, list) and len(content) > 0:
+                        first_item = content[0]
+                        if isinstance(first_item, dict) and first_item.get('is_error'):
+                            has_error = True
+
+                if has_error:
+                    metrics['had_errors'] = True
+
+    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)